ansys/mapdl/core/_commands/database/selecting.py

```diff
@@ -60,6 +60,12 @@ def allsel(self, labt="", entity="", **kwargs):
     The $ character should not be used after the  ALLSEL  command.
 
     This command is valid in any processor.
+
+    Examples
+    --------
+
+    >>> mapdl.allsel()
+
```

Comment:
There's no reason to have these extra lines.  I think one extra line at the bottom is ok (after the example), but not at the top, prior to the example lines.

```suggestion
    >>> mapdl.allsel()
```